README.md

```diff
@@ -1,5 +1,203 @@
-# FCC Test Suite Template
+<a name="readme-top"></a>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  <!-- - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portolio] <a name="about-project"></a>
+
+This project entails a product landing page that displays different products , Their description and cost . The site also has a section where it shows some feautures of the company which includes fast shipping , premium materials and quality assurance.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<!-- 
+> Describe the tech stack and include only the relevant sections that apply to your project. -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md">Linter.yml</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Version Control </summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/git_github_basics.md">Git & Github</a></li>
+  </ul>
+</details> 
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- **[Includes_a_descriptive_readme_file]**
+- **[Has_Client_side_rendering]**
+- **[It_is_a_simple_application]**
```

Comment:
The features of your project should be more descriptive , _Not everyone know the meaning of client-side rendering_

>